pydicom/config.py

```diff
@@ -68,20 +68,31 @@ def DS_decimal(use_Decimal_boolean=True):
 logger.addHandler(handler)
 
 
-image_handlers = []
-"""Image handlers for converting pixel data.
-This is an ordered list that the dataset._get_pixel_array()
+import pydicom.pixel_data_handlers.numpy_handler as np_handler  # noqa
+import pydicom.pixel_data_handlers.rle_handler as rle_handler  # noqa
+import pydicom.pixel_data_handlers.pillow_handler as pillow_handler  # noqa
+import pydicom.pixel_data_handlers.jpeg_ls_handler as jpegls_handler  # noqa
+import pydicom.pixel_data_handlers.gdcm_handler as gdcm_handler  # noqa
+
+pixel_data_handlers = [
+    np_handler,
+    rle_handler,
+    gdcm_handler,
+    pillow_handler,
+    jpegls_handler,
+]
```

Comment:
I think we should alias and give deprecation warning for the old name `image_handlers`.  It's possible users could have used it.

Reply:
The behaviour is different though. `image_handlers` would include only those handlers that were available given installed dependencies while `pixel_data_handlers` includes all handlers. If you think an alias is the best approach then that's OK with me.

Reply:
I'll have to look at that again, I thought this was just a rename of image_handlers.  `pixel_data_handlers` is a more accurate name as technically not all PixelData is an image in the usual sense (e.g. RT dose).

Reply:
Also I'd like to add support for OverlayData and WaveformData at some point

Reply:
Just to clarify, currently we try to import the handlers but if they raise an exception (usually `ImportError` due to missing dependencies) then they don't get added to `image_handlers`. This PR (and also the older conflicted one) changes that so that `ImportErrors` are ignored and all handlers added to `pixel_data_handlers`. Its then up to whatever uses the handlers to determine if they can be used by checking the `handler.is_available` method.

Reply:
If you're happy with this approach then I should probably update the changelog with a bit more of an explanation of the changes than what I've currently got.